samples/matter/light_bulb/src/app_event.h

```diff
@@ -10,45 +10,38 @@
 
 #include "led_widget.h"
 
-struct AppEvent {
-	enum LightEventType : uint8_t { On, Off, Toggle, Level };
-
-	enum FunctionEventType : uint8_t { FunctionPress = Level + 1, FunctionRelease, FunctionTimer };
-
-	enum UpdateLedStateEventType : uint8_t { UpdateLedState = FunctionTimer + 1 };
+struct AppEvent;
+typedef void (*EventHandler)(const AppEvent &);
 
-	enum OtherEventType : uint8_t {
-		StartBleAdvertising = UpdateLedState + 1,
-		PublishLightBulbService,
+struct AppEvent {
+	enum AppEventTypes {
```

Comment:
```suggestion
	enum AppEventType {
```
It's rather unusual to name enums in a plural form.